p2p/peer_behaviour_test.go

```diff
@@ -0,0 +1,23 @@
+package p2p
+
+import (
+	"net"
+	"testing"
+)
+
+func TestStorePeerBehaviour(t *testing.T) {
+	peer := newMockPeer(net.IP{127, 0, 0, 1})
+	pb := NewStorePeerBehaviour()
+	pb.Errored(peer, ErrorBehaviourUnknown)
+
+	peerErrors := pb.GetErrored()
```

Comment:
For some extra safety we could check if the returned list has at least one element. Same for the call below.